weblate-scripts/pre_commit.py

```diff
@@ -72,14 +73,16 @@ def properties_to_xwiki_xml_properties(file_path, path_prefix, base_file_name, l
     base_properties = PropertiesFile()
     base_properties.load(content)
 
-    # Replace keys with the current translation
-    base_properties.replace_with(properties)
-    base_properties.filter_export()
+    if not properties.is_empty():
```

Comment:
Same as above, and you will be able to do the check earlier.

Reply:
hm indeed I mixed up between base_properties and properties. I'll check it above